ibft/instance.go

```diff
@@ -28,6 +28,7 @@ type InstanceOptions struct {
 	ValueCheck     valcheck.ValueCheck
 	LeaderSelector leader.Selector
 	Params         *proto.InstanceParams
+	PublicKey      []byte
```

Comment:
no need to add it, we already have validatorPK

Reply:
with that we populating the state pk

Reply:
but we already have validator PK it's just a duplicate